scripts/mkcloud

```diff
@@ -452,35 +452,12 @@ function onhost_enable_ksm
 
 function setuphost()
 {
-    if is_suse ; then
-        export ZYPP_LOCK_TIMEOUT=60
-        kvmpkg=kvm
-        osloader=
-        ipxe=
-        [[ $arch = aarch64 ]] && {
-            kvmpkg=qemu-arm
-            osloader=qemu-uefi-aarch64
-            ipxe=qemu-ipxe
-        }
-        [[ $arch = s390x ]] && kvmpkg=qemu-s390
-        zypper --non-interactive in --no-recommends \
-            libvirt $kvmpkg $osloader $ipxe lvm2 curl wget bridge-utils \
-            dnsmasq netcat-openbsd ebtables libvirt-python
-        [ "$?" == 0 -o "$?" == 4 ] || complain 10 "setuphost failed to install required packages"
-
-        # enable KVM
-        [[ $arch = s390x ]] && {
-            echo 1 > /proc/sys/vm/allocate_pgste
-        }
-    fi
+    do_setuphost
+}
 
-    sed -i 's/net.ipv4.ip_forward = 0/net.ipv4.ip_forward = 1/' /etc/sysctl.conf
-    echo "net.ipv4.conf.all.rp_filter = 0" > /etc/sysctl.d/90-cloudrpfilter.conf
-    echo 0 > /proc/sys/net/ipv4/conf/all/rp_filter
-    if [ -n "$needcvol" ] ; then
-        safely pvcreate "$cloudpv"
-        safely vgcreate "$cloudvg" "$cloudpv"
-    fi
+function do_setuphost()
```

Comment:
Then you got me wrong - or I phrased it wrong.
The goal was that a user calling an mkcloud run manually should still be able to call `mkcloud cleanup` (and the correct thing happens) instead of having to call `mkcloud libvirt_cleanup`.
I don't see a reason to keep the `do_` functions here.

The `do_` functions used to hold the code that does the job where as the non-`do_` function was the api or set some variables or ran a loop which called the `do_` function repeatedly. They are not called as steps, so there is no reason to keep them.